bridge/github/config.go

```diff
@@ -291,14 +495,48 @@ func prompt2FA() (string, error) {
 		fmt.Print("two-factor authentication code: ")
 
 		byte2fa, err := terminal.ReadPassword(int(syscall.Stdin))
+		fmt.Println()
 		if err != nil {
 			return "", err
 		}
 
-		if len(byte2fa) > 0 {
-			return string(byte2fa), nil
+		if len(byte2fa) != 6 {
```

Comment:
I did this because when i tried to test the changes i miss typed the 2FA code and had to do everything again 😆. ( I thought that all 2FA methods are officially 6digits, at least for SMS/and Google authenticator apps) 
Do you think we should give users a second and third chance to type the 2FA code ?

Reply:
Depending on what make sense in the code, you can either repeat the 2FA process or the whole auth process. That said, it's a nice thing to have but remember that it's a process that will be used about once per user. It's acceptable if it's not perfect :-)

Reply:
It is true users will have to do this only once. Let's keep it this way then 👍